rc/base/spell.kak

```diff
@@ -9,6 +9,10 @@ def -params ..1 spell -docstring "Check spelling of the current buffer with aspe
         printf %s\\n "set buffer spell_tmp_file ${file}"
     }
     %sh{
+        _echo() {
```

Comment:
In all the cases where you use _echo, we can just use normal echo safely, as we always know the single argument we pass is not starting with '-', I'd prefer if you removed that _echo function and just used built-in echo.